index.html

```diff
@@ -1,17 +1,37 @@
-Hannah Donnelly
-
-About
-
-Hey, I’m Hannah.
-
-With a degree in Design & Merchandising from Drexel University and years of Food, Beverage & Hospitality work under my belt, I’m one of the great many people out there working hard to make a major career change and move into programming and STEM at large. When I’m not on the clock or studying, I can typically be found watching Iron Chef or playing The Legend of Zelda: Breath of the Wild for the 47th time. 
-
-Experience
-
-I am currently in an introductory programming course through an excellent organization called Code the Dream. Check back for updates!
-
-Connect
-
-Email - mailto:hannah.v.donnelly@gmail.com
-
-GitHub - www.github.com/hvd28
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Home - Hannah Donnelly</title>
+    <meta name="author" content="Hannah Donnelly">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Hannah Donnelly's programming portfolio">
+</head>
+
+<body>
+    <section>
+        <h1>About</h1>
+
+        Hey, I’m Hannah.
+
+        <p>With a degree in Design & Merchandising from Drexel University and years of Food, Beverage & Hospitality work
+            under my belt, I’m one of the great many people out there working hard to make a major career change and
+            move into programming and STEM at large. When I’m not on the clock or studying, I can typically be found
+            watching Iron Chef or playing The Legend of Zelda: Breath of the Wild for the 47th time.</p>
+    </section>
+    <section>
+        <h1>Experience</h1>
+
+        <p>I am currently in an introductory programming course through an excellent organization called Code the Dream.
+            Check back for updates!</p>
+    </section>
+    <section>
+        <h1>Connect</h1>
+        <ul>
+            <li><a href="mailto:hannah.v.donnelly@gmail.com">Email me</a></li>
+            <li><a href="www.github.com/hvd28">Check out my GitHub</a></li>
```

Comment:
```suggestion
            <li><a href="https://github.com/hvd28">Check out my GitHub</a></li>
```

If you don't include the protocol (`http:` or `https:`) the link is _relative_ and would end point pointing to something like `https://yourwebsite.com/github.com/hvcd28`: Not what you intend.